app/javascript/packs/personal-key-page-controller.js

```diff
@@ -125,9 +125,14 @@ function downloadForIE(event) {
   window.navigator.msSaveBlob(blob, filename);
 }
 
+function trackDownload() {
+  trackEvent('IdV: personal key downloaded');
```

Comment:
Sorry to adjust again, but I noticed that in the upcoming reimplementation of this step, this event was implemented in #6257 as ...

```suggestion
  trackEvent('IdV: download personal key');
```

It would be good to keep these in sync so we don't have to worry about querying for two event names.